### DISCUSSION TITLE
Prior Weeks Starting Lineups

### DISCUSSION THREAD

**Discussion by egor006** [cwendt94/espn-api#479] [ESPN API]
I'm trying to determine if a player was started vs on the bench for prior weeks of the season using the below function. Is this information available anywhere?

```
def getTeamRosters(week: int, team_id: int):
    LEAGUE.load_roster_week(week)
    team = LEAGUE.get_team_data(team_id=team_id)
    return team
```

**Comment by cwendt94** [comment]
You can use `league.box_scores` functionality which will give you all of the information. Check out [box_scores](https://github.com/cwendt94/espn-api/wiki/League-Class) section in the main wiki page. You can pass in the week you want to see and then from the matchup get the players slot position which will tell you in they were on the bench or in a starting position.
